packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -58,6 +68,32 @@ class Bet:
     status: BetStatus = BetStatus.UNPROCESSED
     blacklist_expiration: float = -1
 
+    @classmethod
+    def from_gamma_subgraph(cls, **kwargs: Any) -> "Bet":
+        """Initialize a bet's instance from gamma subgraph's attributes."""
+        outcomes = eval(kwargs["outcomes"])
+        outcome_token_amounts = [0] * len(outcomes)
+        end_date = datetime.strptime(kwargs["endDate"], "%Y-%m-%dT%H:%M:%S.%fZ")
+        opening_timestamp = int(end_date.replace(tzinfo=timezone.utc).timestamp())
+        created_by = kwargs["createdBy"] or NULL_ADDRESS
```

Comment:
```suggestion
        created_by = kwargs["submitted_by"] or NULL_ADDRESS
```

Also replace this field to the query.

Reply:
87eb59cf5020ad920a53f0f97a9e3924e6c3c4cc